packages/grid/_modules_/grid/components/menu/GridMenu.tsx

```diff
@@ -39,14 +39,24 @@ export const GridMenu: React.FC<MenuProps> = ({
   position,
   ...other
 }) => {
+  const targetRef = React.useRef(target);
+  const openRef = React.useRef(open);
+
+  React.useEffect(() => {
+    if (openRef.current && targetRef.current) {
+      (targetRef.current as HTMLElement).focus();
+    }
+
+    openRef.current = open;
+    targetRef.current = target;
+  }, [open, target]);
+
   return (
     <Popper open={open} anchorEl={target as any} transition placement={position} {...other}>
       {({ TransitionProps, placement }) => (
         <Grow {...TransitionProps} style={{ transformOrigin: transformOrigin[placement] }}>
           <Paper>
-            <ClickAwayListener onClickAway={onClickAway}>
-              <div>{children}</div>
-            </ClickAwayListener>
+            <ClickAwayListener onClickAway={onClickAway}>{children}</ClickAwayListener>
```

Comment:
The wrapping div is important, for the logic to work. You should see a prop type warning when you did the change or even the column menu failing to open.

<img width="508" alt="Capture d’écran 2021-02-15 à 16 09 34" src="https://user-images.githubusercontent.com/3165635/107963248-376cd400-6fa8-11eb-953e-2da909c84a35.png">

Reply:
Restored